system/forms/preside-objects/page/edit.xml

```diff
@@ -11,7 +11,7 @@ This form is used as the base "edit page" form for Sitetree pages. See also [[fo
 		<fieldset id="main" sortorder="10">
 			<field sortorder="5" binding="page.parent_page" control="sitetreePagePicker" required="true" />
 			<field sortorder="10" binding="page.title" />
-			<field sortorder="20" binding="page.slug" required="true" />
+			<field sortorder="20" binding="page.slug" required="true" control="autoslug"/>
```

Comment:
This is deliberately *not* an auto slug so that the page URL is not unintentionally changed with changes to the page title. I don't think we should change this behaviour.